dialer.go

```diff
@@ -112,7 +116,8 @@ type dialer struct {
 	cipherCode            Cipher
 	serverPublicKey       *rsa.PublicKey
 	muNumLivePending      sync.Mutex
-	numLivePending        int
+	numLive               int
```

Comment:
Have to break into two variables as we should only compare the number of live sessions when returning session to the channel.